string beacon_id_req
---
string beacon_id
